docs/synthetics/status-pages.md

```diff
@@ -33,6 +33,41 @@ Every status page will have a unique public URL in this format:
 
 You can use this URL to preview the status page before opening to the public. Disabling the status page will block the public access for the status page.
 
+## Edit Status Pages
+
+To edit a status page simply click on the **“...”** that appears when you hover over a status page name on the side bar. Then click on Edit from the menu.
+All the monitors that are already added to the status page will be pre-selected. 
+You can **filter included/not included** monitors by clicking on the buttons next to the search bar.
+
+![Edit Status Page](../images/synthetics/status-page-edit.png)
+
+# Incidents
+
+Incidents lets you inform your users via status pages. This is useful when there is planned maintenance or downtime in your system. 
```

Comment:
Incidents lets you inform your users via status pages. This is useful when there is planned maintenance or downtime in your system. => Incidents let you inform your users of planned maintenance, downtime, or any other issues your service is experiencing.